### DISCUSSION TITLE
After import of Kibana dashboards used in SO 2.3, kibana dashboards 2.4 lenses do not show data

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#11551] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

26

### Storage for /

200 gb

### Storage for /nsm

1 tb

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello all,

After the import of Kibana dashboards and lenses used in SO 2.3, and deleting them after I concluded that they do not work, I see several lenses with the remark "Could not locate that visualization (id: d04b5130-6e99-11ea-9266-1fd14ca6af34)". I think that these lenses are overwritten by my import.

I have already tried to repair this with the following commands:

- sudo so-docker-refresh
- sudo so-kibana-config-load
- sudo salt-call state.apply kibana.so_savedobjects_defaults -l info queue=True

But this did not help.

Is there a command to repair these lenses and searches?

Regards

Bart

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by cm-ops** [comment]
Try the following: 
```
Reload the Kibana config:
sudo rm /opt/so/state/kibana_*.txt
sudo salt-call state.apply kibana.so_savedobjects_defaults -l info queue=True
```

**Reply by sleepingbel** [reply]
Thx It works
